tonic_validate/metrics/retrieval_precision_metric.py

```diff
@@ -12,6 +12,13 @@
 class RetrievalPrecisionMetric(Metric):
     name: str = "retrieval_precision"
 
+    def __init__(self):
+        """
+        Metric that checks whether the context retrieved is relevant to answer the given question.
```

Comment:
Metric that checks whether the retrieved context is relevant to answer the given question.